libraries/AP_HAL_ChibiOS/hwdef/BETAFPV-F405/hwdef.dat

```diff
@@ -53,27 +53,33 @@ SERIAL_ORDER OTG1 USART1 EMPTY USART3 UART4 EMPTY USART6
 # PA10 IO-debug-console
 PA11 OTG_FS_DM OTG1
 PA12 OTG_FS_DP OTG1
+PC5 VBUS INPUT OPENDRAIN
 
 # USART1 (GPS)
 PA10 USART1_RX USART1
 PA9 USART1_TX USART1
 define DEFAULT_SERIAL1_PROTOCOL SerialProtocol_GPS
 
-define DEFAULT_SERIAL2_PROTOCOL SerialProtocol_None
-define DEFAULT_SERIAL5_PROTOCOL SerialProtocol_None
-
 # USART3 (ELRS)
 PB10 USART3_TX USART3
 PB11 USART3_RX USART3
 define DEFAULT_SERIAL3_PROTOCOL SerialProtocol_RCIN
 
-# UART4 (DJI)
+# UART4 (MSP)
 PA0 UART4_TX UART4 NODMA
 PA1 UART4_RX UART4 NODMA
-define DEFAULT_SERIAL4_PROTOCOL SerialProtocol_DJI_FPV
+define DEFAULT_SERIAL4_PROTOCOL SerialProtocol_MSP_DisplayPort
+
+# UART5 (SBUS, RX-only)
```

Comment:
is this inverted to RX5? wont work otherwise

Reply:
It's labelled as SBUS on the connector. It's either hardware inverted or uses the inversion pin (which is already defined)